linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_renameat2/rule.yml

```diff
@@ -43,6 +43,18 @@ warnings:
         -a always,exit -F arch=b64 -S renameat2 -F auid>=1000 -F auid!=4294967295 -k perm_mod</pre>
 
 template:
+    {{% if product not in ["sle15"] %}}
```

Comment:
Given that `prodtype` contains only `sle15` I don't understand what's the reason for this if and else.

Reply:
My apologies, I was on vacation I will make the change said above. :)

Reply:
It seems another PR was already made by someone else to deal with this so I will just close this one :)